ndk gradle project (#18)
getsentry/sentry-android: sentry-android-ndk/CMakeLists.txt

cmake_minimum_required(VERSION 3.4.1)
project("sentry-android")

> # Adding NDCrash submodule subdirectory

Reviewer: ```suggestion
# Adding sentry-native submodule subdirectory
```

add_subdirectory("sentry-native")

> # NDCrash submodule include path.

Reviewer: ```suggestion
# sentry-native submodule include path.
```

include_directories("sentry-native/include")

> # JNDCrash source code.

Reviewer: ```suggestion
# sentry-native source code.
```

file(GLOB SENTRY_ANDROID_SOURCES ${CMAKE_SOURCE_DIR}/src/main/jni/*.c)

> # Adding dynamic library and linking it with log and NDCrash.

Reviewer: ```suggestion
# Adding dynamic library and linking it with log and sentry-native.
```
